apps/condo/domains/routes/paymentLinkRouter.js

```diff
@@ -93,6 +94,12 @@ class PaymentLinkRouter {
     }
 
     async handleRequest (req, res) {
+        const isEnabled = await featureToggleManager.isFeatureEnabled(this.context, 'payment_link')
```

Comment:
the context should be unique for each request. Is it true?

Reply:
It is not unique for each request